.github/workflows/build.yml

```diff
@@ -0,0 +1,87 @@
+name: "Build and test Malwarecage"
+on:
+  push:
+    branches:
+    - master
+    - ci/*
+  pull_request:
+    branches:
+    - master
+jobs:
+  build_core:
+    name: Build Malwarecage core image
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Build and push Malwarecage image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
FWIW, secrets can be leaked by any user with write access to the repository. Can we use a dockerhub token here?

Reply:
Good point. Docker Hub tokens are used as password, so it doesn't need any change in workflow file.